frontend/pages/index.tsx

```diff
@@ -92,6 +100,31 @@ export default function Home(): JSX.Element {
           );
         })}
       </div>
+
+      <Flex direction="column" align="center" width="lg">
+        <Heading mb={2} align="center" colorScheme="gray">
+          {formatMessage(
+            { id: "heroHeading" },
+            {
+              span: function renderSpan(...parts) {
+                return (
+                  <Box as="span" color="blue.500">
+                    {parts.join("")}
+                  </Box>
+                );
+              },
+            }
+          )}
```

Comment:
Awesome work on this! Love the flexibility it brings to the component, should be a great use for i8n!